src/qml/editorwidgets/TextEdit.qml

```diff
@@ -29,7 +29,9 @@ EditorWidgetBase {
     color: Theme.mainTextColor
     opacity: 0.45
     wrapMode: Text.Wrap
-    textFormat: config['IsMultiline'] === true && config['UseHtml'] ? TextEdit.RichText : TextEdit.AutoText
+    textFormat: (config['IsMultiline'] === true && config['UseHtml']) || stringUtilities.hasLinks(value)
```

Comment:
Why to you use `=== true` for `IsMultiline` but not for `UseHtml`?

Reply:
@domi4484 , good question, it's been like that forever and not inclined to touch the delicate equilibrium here :)